[INFO] Module GCC/9.4.0 loaded.

Inactive Modules:
  1) UCX/1.17.0     2) numactl/2.0.16

The following have been reloaded with a version change:
  1) GCCcore/.11.3.0 => GCCcore/.9.4.0     3) zlib/1.2.13 => zlib/1.2.11
  2) binutils/2.38 => binutils/2.36.1

[INFO] Module CUDA/11.8.0 loaded.
[INFO] Module cuDNN/8.6.0.163-CUDA-11.8.0 loaded.
Traceback (most recent call last):
  File "/home/md334167/elzerman-with-unet/train_for_exdata.py", line 358, in <module>
    main()
  File "/home/md334167/elzerman-with-unet/train_for_exdata.py", line 307, in main
    axs[0].plot(x[i].reshape(-1), label='Noisy', color='mediumblue', linewidth=0.9)
                ~^^^
IndexError: index 32 is out of bounds for axis 0 with size 32
